pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -863,6 +917,8 @@ public CmdFunctions(PulsarAdmin admin) throws PulsarClientException {
         uploader = new UploadFunction();
         downloader = new DownloadFunction();
         jcommander.addCommand("localrun", getLocalRunner());
+        jcommander.addCommand("k8run", getK8Runner());
```

Comment:
should the k8 operations be under a sub category e.g.

./pulsar-admin functions k8 create/delete